Packages/MovieDBUI/Sources/MovieDBUI/UI/Components/MovieCell/MovieTableViewCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  MovieTableViewCell.swift
+//  MovieDB
+//
+//  Created by Mouad Bj on 16/3/2023.
+//
+
+import UIKit
+import MovieDBCore
+
+struct MovieCellViewModel: Hashable {
+  var movie: Movie
+  var posterURL: URL?
+}
+
+class MovieTableViewCell: UITableViewCell {
```

Comment:
Yes, That would be even better IMO, as it will make the system even more modular. but we need to group/name them in a certain way that still shows to which layer every package belongs.